### DISCUSSION TITLE
Cannot show your support without agreeing that every RMS's argument was `reasonable`

### DISCUSSION THREAD

**Discussion by zaytsevand** [rms-support-letter/rms-support-letter.github.io#6258] [General]
I adore efforts that were made to keep the text unbiased for the most part, but the line:
> To the ambush mob who is ganging up on Richard Stallman over reasonable arguments in debate and various opinions and beliefs voiced over decades as a public figure

implies a valuation of RMS's arguments one shouldn't be able to make without a thorough research. Which is, frankly, doesn't seem to be possible.

Also, by itself
> the ambush mob who is ganging up on

is a bit too emotionally colored as far as I'm concerned.

1st issue could be fixed by removing "reasonable" part.
2nd issue could be more of a personal taste. And I cannot raise and argument that this formula do not reflect the situation.

**Comment by 6r1d** [comment]
> implies a valuation of RMS's arguments one shouldn't be able to make without a thorough research. Which is, frankly, doesn't seem to be possible.

You can:

* Discuss what was written and said
* Generally learn more
* Analyze emotions of people mentioning stuff and their vested interests
* Use Google / DuckDuckGo / etc.
* Analyze open letter arguments one by one
* Read issues here, people did and found many refutals
* Read RMS comments, because sometimes, he actually addresses things
* Use Internet Archive to track records and changes
* Use Wikipedia logs to track edits (people did)

From that point on, you may check how credible the evidence you have is. Both for arguments for and arguments against.
And do research from there. There's nobody standing behind your back and stopping you.

> 1st issue could be fixed by removing "reasonable" part.

I have a **hypothesis**, but not more. Imagine you know someone ([Marvin](https://www.youtube.com/watch?v=EI0NXTrS5Pw)) well. And know he's old, frail, peaceful guy, who travels with his wife. You're confident and write random stuff on your mind, because you don't expect the widespread reaction. You don't do a full analysis, you don't request information from others, you don't gather evidence: you expect a bunch of people arguing over it and getting information. Your phrasing offends people and they make aggressive conclusions about you.

They do, because people [tend to](https://theoatmeal.com/comics/believe). There are theories for vested interest in whole conversation, as well: corporations aren't super happy about GNU GPL.

> is a bit too emotionally colored as far as I'm concerned.

If we were **able to** have a discussion about RMS openly and politely, none of what you see would happen.

We attempted to [establish a conversation](https://github.com/rms-support-letter/rms-support-letter.github.io/issues/175) initially and it failed, many, many times, for what it's worth.

**Reply by zaytsevand** [reply]
I'm sorry, but a research you've suggested isn't feasible. Not for me, not for many who have signed nor will sign your letter. And I think we can both agree on it. As reasonable persons.

Tossing "reasonable" casually into a sentence is the same kind of trick that is used by corporations when they use small font or footnotes to hide unfavorable conditions. People will likely to swallow it without giving a second thought.

I've tried to have an open discussion, and was denied. Issue have a superior visibility over discussions, and handful of issues are discussed as they are. Even if they stress far less important matters. And I deem a misguidance that I've pointed on as a very important one.

But nevertheless we can discuss it here and dive into the understanding of morale, liberty and freedom, if you wish.

**Reply by 6r1d** [reply]
> But nevertheless we can discuss it here and dive into the understanding of morale, liberty and freedom, if you wish.

Oh well. If you aren't interested in looking for information, which is available, it all boils down to two legal approaches.

* Guilty until proven innocent. Accuse RMS, force him to protect himself, **because someone told so**.
* Innocent until proven guilty. Accept he's a flawed human being and leave him be, if someone really thinks he's some evil person, they need proof.

I'd side with "innocent until proven guilty" approach: an open letter means there are no proofs, just a lot of people who may or may not have vested interests. GPL license Richard wrote is a barrier for corporations in many regards, so it's easy to expect thousands people willing to dismantle it any single day.

-----

> I've tried to have an open discussion, and was denied. Issue have a superior visibility over discussions

Oh, and there is a **big difference**. This discussion stays. Issues on rms-open-letter [get closed or deleted](https://github.com/rms-support-letter/rms-support-letter.github.io/issues/175). And supporting RMS is a wider topic than a single dialog.

**Reply by zaytsevand** [reply]
No that's not like that. In a scientific world at least. If you're making a statement you must have arguments to back it up, if requested. This is what science is essentially.

So if you're (and I, as a signer of this letter) making a statement that every and each argument ever presented by mr. Stallman was reasonable, we've to back that claim up. Otherwise we're backing it with our numbers purely, which is EXACTLY what is wrong with the campaign going against mr. Stallman. Undermining our declaration of sticking to facts over public pressure.

**Reply by zaytsevand** [reply]
Also, as I've stated previously, by putting this statement in the letter we're tricking signers to agree with something that have an ambiguous interpretation. And it's simply wrong. As wrong, as EULAs with footnotes.

**Reply by nukeop** [reply]
> No that's not like that. In a scientific world at least. If you're making a statement you must have arguments to back it up, if requested. This is what science is essentially.

Ok, but this isn't science.

> Also, as I've stated previously, by putting this statement in the letter we're tricking signers to agree with something that have an ambiguous interpretation. And it's simply wrong. As wrong, as EULAs with footnotes.

It's pretty clear what it says. It's short, to the point, and we're not hiding anything. If you don't agree, you don't have to sign. Changing the contents of the letter after ~6000 people signed it is not an option, it would invalidate all the signatures we have already collected.

**Reply by zaytsevand** [reply]
@nukeop so you've chosen to employ same tactics of unprovable offensive public claims, as RMS's attackers. Lost before the fight even began, sorry. Retracting my signature.
